gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-2-entrypoints/step-2-entrypoints-2-config.harness.ts

```diff
@@ -42,6 +44,10 @@ export class Step2Entrypoints2ConfigHarness extends ComponentHarness {
 
   protected getListenersDiv = this.locatorFor('#listeners');
 
+  // protected getQosSelect = (entrypointId: string) => this.locatorFor(GioFormQosHarness);
```

Comment:
Remove comment

Reply:
🤦🏻 forgot to cleanup this during my tests